### PR TITLE
Fix loop during unit tests between std.parallelism and std.range

### PR DESCRIPTION
**Pull request by schveiguy**
Hopefully this fixes the cycle detected in dlang/druntime#1602

### CODE REVIEW

**Comment by JackStouffer** [comment]
Yeah, this should have been in `parallelism.d` in the first place.

LGTM

**Comment by schveiguy** [comment]
ping @klickverbot need this merged to continue with cycle detection fix.

**Comment by DmitryOlshansky** [comment]
Auto-merge toggled on

File: std/parallelism.d [hunk 1]
```diff
@@ -4561,3 +4561,9 @@ unittest
 
     auto result = taskPool.amap!__genPair_12733(data);
 }
+
+unittest
+{
+    // this test was in std.range, but caused cycles.
```
Comment (MartinNowak) [inline std/parallelism.d]: What cycles you mean remains unclear in this context.

**Pull request merged**